docker-setup.sh

```diff
@@ -6,12 +6,13 @@ set -e
 
 pull_docker_snapshot() {
   project="${1?project name required}"
+  stack_version="$2"
   local docker_image="docker.elastic.co/${project}/${project}${DISTRIBUTION_SUFFIX}:${ELASTIC_STACK_VERSION}"
   echo "Pulling $docker_image"
   if docker pull "$docker_image" ; then
     echo "docker pull successful"
   else
-    case "$ELASTIC_STACK_VERSION_ARG" in
+    case $stack_version in
```

Comment:
interesting. this pr changes how this function accesses the original `ELASTIC_STACK_VERSION` value, but the pre-existing funciton only _uses_ the value to exit with a different status code when that value is in a list of specific aliases?

I don't see the exit status code [being observed](https://github.com/logstash-plugins/.ci/pull/66/files#diff-228a8300e10dffb4f00f6a2231dca4544c95c6ca4216f93cc4beaabf70db8929R67) though. I'm not sure what's going on here, or why we have it.

Reply:
I think the exit status is used by `exec.yml` (as of very recently) https://github.com/logstash-plugins/.ci/commit/15493f15654271070529b8e4614963147732cd0e